armi/physics/thermalHydraulics/parameters.py

```diff
@@ -134,13 +93,6 @@ def _getBlockParams():
             categories=["thInterface"],
         )
 
-        pb.defParam(
-            "THdeltaPBundle",
-            units="Pa",
-            description="Pressure difference in a bundle, including contributions from friction, acceleration, and gravity",
-            categories=["thInterface"],
-        )
-
```

Comment:
Arguably we can remove the other TH parameters too because these are also specific to our internal plugins. While these can be used by other folks it's more likely that someone who makes a TH plugin would just reimplement their own parameters. 

I think that in general we probably over committed to the number of parameters that are available on the vanilla ARMI framework and they probably don't have a lot of general use cases.

Reply:
There might be other TH parameters we can use.

But they will all be more complicated, because they are used by multiple downstream projects.

These particular TH parameters are the easiest for me to vet, because they are only used by the one downstream repo.

Low hanging fruit.